py3-bokeh.yaml

```diff
@@ -38,8 +107,6 @@ pipeline:
   - name: Python Build
     uses: python/build-wheel
 
-  - uses: strip
-
```

Comment:
i think we're leaving these in. they don't do anything if there aren'[t c modules, but its not harmful (and if it was, we need to fix it).